Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -1514,6 +1519,14 @@ - (NSTimeInterval)age {
     return -[[self creationDate] timeIntervalSinceNow];
 }
 
+- (BOOL)isSymlink {
+    if (self.fileAttributes[NSFileType] == NSFileTypeSymbolicLink) {
```

Comment:
This could be simplified to directly `return self.fileAttributes[NSFileType] == NSFileTypeSymbolicLink;`.

Reply:
Done, thanks